src/test/resources/org/prebid/server/it/openrtb2/algorix/test-auction-algorix-request.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "tid",
+  "imp": [
+    {
+      "id": "impId001",
+      "banner": {
+        "format": [
+          {
+            "w": 300,
+            "h": 250
+          }
+        ]
+      },
+      "ext": {
+        "algorix": {
+          "sid": "testSid",
+          "token": "testToken"
+        }
+      }
+    }
+  ],
+  "device": {
+    "pxratio": 4.2,
+    "dnt": 2,
+    "language": "en",
+    "ifa": "ifaId"
+  },
+  "site": {
+    "publisher": {
+      "id": "publisherId"
+    }
+  },
+  "at": 1,
+  "tmax": 5000,
+  "cur": [
+    "USD"
+  ],
+  "source": {
+    "fd": 1,
+    "tid": "tid"
+  },
```

Comment:
Can be removed, we prefer to keep tests as simple as possible

Reply:
I removed this code, but in Java CI / build (8) (pull_request) failed.
this is reply:
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    AlgorixTest.openrtb2AuctionShouldRespondWithBidsFromAlgorix:57 ext.responsetimemillis
Expected: cache
      but none found
  ; ext
 Unexpected: errors
  ; seatbid[]: Expected 1 values but got 0 ; 
 Unexpected: nbr

[INFO] 
Error:  Tests run: 3700, Failures: 1, Errors: 0, Skipped: 0

but I run it in local is success.
I see other adapter this code not removed.